style.css

```diff
@@ -0,0 +1,861 @@
+}
+
+.black-bar {
+  width: 20px;
+  height: 3px;
+  background-color: rgb(202, 189, 189);
+  margin-bottom: 8px;
+}
+
+.description-feature {
+  margin-left: 20px;
+}
+
+.who-man {
+  font-family: poppins-regular, sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.description-feature h5 {
+  font-family: poppins-bold, sans-serif;
+  font-size: 20px;
+  font-weight: 600;
+  padding: 5px;
+}
+
+.description-feature h6 {
+  font-family: poppins-regular, sans-serif;
+  font-size: 16px;
+  color: red;
+}
+
+.partner-row {
+  background-color: black;
+}
+
+.partner-row h3 {
+  margin-top: 30px;
+  color: #fff;
+  text-align: center;
+  font-family: poppins-regular, sans-serif;
+  font-size: 30px;
+  font-weight: 400;
+  line-height: 25px;
+}
+
+.partner {
+  width: 100%;
+  height: auto;
+  display: inline-flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.col-sm img {
+  width: 100px;
+  height: 40px;
+  margin: 10px;
+  border-radius: 8px;
+}
+
+.footer {
+  width: 95%;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  margin-left: 10px;
+}
+
+.footer img {
+  width: 200px;
+  margin-right: 40px;
+}
+
+.footer h6 {
+  font-family: poppins-regular, sans-serif;
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 25px;
+}
+
+.footer p {
+  font-family: poppins-regular, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 20px;
+}
+
+.home-about {
+  margin-top: 70px;
+  position: relative;
+  text-align: center;
+  font-family: poppins-regular, sans-serif;
+
+  /* border-bottom: 2px solid #656565; */
+}
+
+.home-about h3 {
+  font-size: 24px;
+  font-weight: 700;
+  font-family: poppins-regular, sans-serif;
+}
+
+.home-about .black-bar {
+  width: 50px;
+  margin-left: auto;
+  margin-right: auto;
+  height: 5px;
+  border-radius: 5px;
+  background-color: rgb(238, 66, 66);
+}
+
+.home-about p {
+  font-size: 16px;
+  font-weight: 500;
+  width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 20px;
+}
+
+.home-about img {
+  width: 30%;
+  height: 30%;
+  margin: 20px;
+}
+
+.about-footer {
+  background-image: url(images/bg-image.jpg);
+}
+
+.about-footer h6 {
+  font-size: 16px;
+  color: #fff;
+  font-weight: 700;
+}
+
+.about-footer p {
+  font-style: italic;
+  color: #fff;
+  font-size: 12px;
+}
+
+.home-image {
+  margin-top: 20px;
+}
+
+.home-image .row-1 {
+  margin-left: auto;
+  margin-right: auto;
+  display: inline-flex;
+  justify-content: center;
+  align-self: center;
+}
+
+.home-image .col-sm6 img {
+  margin: 10px;
+  width: 200px;
+  height: 200px;
+  opacity: 0.7;
+}
+
+.about-1 {
+  border-bottom: 1px solid #656565;
+}
+
+.Btn-text {
+  display: none;
+}
+
+.disNone {
+  display: flex;
+}
+
+.active a {
+  color: #e61a1a;
+  font-weight: 800;
+}
+
+a.Border-nav {
+  color: #e61a1a;
+  font-weight: 700;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Please make sure your CSS styling is built as mobile-first, media queries are used to change the design from mobile to desktop, not the other way around. Kindly use a media query with `min-width` instead and put all your desktop styling in the media query and take all your mobile styling out of the media query.